libs/openFrameworks/utils/ofUtils.h

```diff
@@ -735,11 +735,11 @@ char ofToChar(const string& charString);
 template <class T>
 string ofToBinary(const T& value) {
 	ostringstream out;
-	const char* data = (const char*) &value;
+	const unsigned long long* data = static_cast<unsigned long long*>(&value);
```

Comment:
Yeah, it's the default argument for the bitset, so it definitely gets rid of warnings.  I'll update to the more compact type.